src/src/components/obComponents/grids/obVuetifyGrid.vue

```diff
@@ -73,4 +227,73 @@
     async function search(): Promise<void> {
         entityGrid.value.search = searchText.value;
     }
+
+    async function addFilter(type: FilterType, key: string, value: string | number | null) {
+        props.entityGrid.extraFilters = props.entityGrid.extraFilters.filter((element) => element.key != key);
+        if (value != null) {
+            var filter = new Filter(type, key, value);
+            props.entityGrid.extraFilters.push(filter);
+        }
+        props.entityGrid.refresh();
+    }
+
+    async function addDateFilter(
+        type: FilterType,
+        key: string,
+        value: Date | null,
+        isDateTimeOffset: boolean = false,
+    ): Promise<void> {
+        props.entityGrid.extraFilters = props.entityGrid.extraFilters.filter((element) => element.key != key);
+        if (value != null) {
+            var filter = new Filter(type, key, value, null, isDateTimeOffset);
+            props.entityGrid.extraFilters.push(filter);
+        }
+        props.entityGrid.refresh();
+    }
+
+    async function addComplexFilter(
+        type: FilterType,
+        key: string,
+        value: string | number | [] | null,
+        operator: StringOperators | NumberOperators | null,
+    ): Promise<void> {
+        props.entityGrid.extraFilters = props.entityGrid.extraFilters.filter((element) => element.key != key);
+        if (value != null) {
+            var filter = new Filter(type, key, value, null, false, operator);
```

Comment:
`var` is generally a bad idea in javascript - `let` is a safer choice 99% of the time. Or even better, `const` can be used here. Looks like there are quite a few other uses of `var` in this PR, please change them to `const` (or `let` if they have to be variable)